server/api.js

```diff
@@ -2,11 +2,40 @@ import { Router } from "express";
 
 import logger from "./utils/logger";
 
+import db from "./db";
+
 const router = Router();
 
 router.get("/", (_, res) => {
 	logger.debug("Welcoming everyone...");
 	res.json({ message: "Hello, world!" });
 });
 
+router.get("/energizers", async (_, res) => {
+	logger.debug("Welcoming everyone...");
```

Comment:
Don't think this is necessary, if anything you could be logging the output of the query to debug?

Reply:
I had no exposure to loger.debug and was just following the existing code to make sure my code works